app/src/docs/content/core/sharingResourcesInCore.mdx

```diff
@@ -0,0 +1,230 @@
+import SvgIcon from '$shared/components/SvgIcon'
+import cx from 'classnames'
+
+import Image from '$docs/components/Image'
+
+import ShareSettings from './images/sharing_resources_in_core_02_desktop.jpg'
+import ShareSettings2x from './images/sharing_resources_in_core_02_desktop@2x.jpg'
+import Sharing from './videos/sharing_resources_in_core_01.mp4'
+
+import docsStyles from '$docs/components/DocsLayout/docsLayout.pcss'
+import coreStyles from '$docs/components/Pages/Core/core.pcss'
+
+# Sharing resources in Core
+
+When using the Core app you can share most resources with other Streamr user accounts. Streams, canvases and dashboards can all be shared. There are two routes in Core for sharing resources, from the edit view or the main display view for that resource. Streams are shared from the list item menu or the share button in the edit view. Canvases and
+dashboards are shared from the tile menu or the share button on the editor toolbar.
+
+<figure>
+    <div className={coreStyles.videoFrame}>
+        <video muted loop autoPlay>
+            <source src={Sharing} type="video/mp4" />
+        </video>
+    </div>
+</figure>
+
+## The share sidebar
+
+Wherever you access it from, all resources are shared from the sidebar. The general flow 
+of sharing with the sidebar works like this.
+
+1. Choose private or public resource
+2. Type the address of the person you want invite to share (either email or Ethereum)
```

Comment:
Are we still including emails?

Reply:
No, I guess that was just old copy... changed it to say `Type the Ethereum address of the person you want invite to share`